modules/ossm-threescale-integration-settings.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/threescale-adapter.adoc
+
+[id="ossm-threescale-integration-settings_{context}"]
+= Configure the integration settings in 3scale
+
+[NOTE]
+====
+For 3scale SaaS customers, {ProductName} is enabled as part of the Early Access program.
+====
+
+.Procedure
+
+. Navigate to *[your_API_name] > Integration > Configuration*.
```

Comment:
Each click in the UI needs to be enclosed in *, like `*Operators* -> *Catalog Sources*`.

Reply:
Fixed.